in/unit/push-to-cloud.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=All-of-Us Arrays Push-to-Cloud Daemon
+After=network.target
+StartLimitIntervalSec=0
+
+[Service]
+EnvironmentFile=-/etc/sysconfig/push-to-cloud
```

Comment:
This path doesn't really look like any of the standard user-level unit config paths.
https://wiki.archlinux.org/index.php/Systemd/User#Environment_variables

Reply:
This will work when the software is installed as an rpm and run as system level service.  The "-" allows the file to not exist so it should still work as a user level service.